RecoLocalTracker/Phase2TrackerRecHits/src/Phase2StripCPEGeometric.cc

```diff
@@ -0,0 +1,28 @@
+
+#include "RecoLocalTracker/Phase2TrackerRecHits/interface/Phase2StripCPEGeometric.h"
+#include "SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerFwd.h"
```

Comment:
@smuzaffar @degano 
is the dependency checker available to be added to the jenkins tests?
Checks for things like ```include Sim``` in ```Reco*/*/{interface,src,plugins} ``` files could be spotted with automated checks.

I see that in this case there is no .o from Sim\* behind this dependency, just not very well placed typedefs include